java/ql/src/change-notes/2023-05-02-post-unicode-normalization-query.md

```diff
@@ -0,0 +1,4 @@
+---
+category: newQuery
+---
+* Added a new query, `java/post-unicode-normalization`, to detect a misuse of a post-unicode normalization.
```

Comment:
(also applies to your Go PR) -- you don't need a change note for an experimental query

Reply:
Fixed in https://github.com/github/codeql/pull/12995/commits/9836cda854749c3a8f976ace363de896a843dd30